### DISCUSSION TITLE
when send write-commond with device-modbus-rtu, errors appears

### DISCUSSION THREAD

**Discussion by Liguangwei0321** [edgexfoundry/community#226] [Q&A]
code: 500 , message: request failed, status code: 500, err: {"apiVersion":"v3","message":"error writing DeviceCommand CS0 state for Modbus-RTU-IO-Module -\u003e handle write command request failed, err: modbus: exception '1' (illegal function), function '143'","statusCode":500}

can i help me

**Comment by jiekechoo** [comment]
@Liguangwei0321 
Please move this issue to [edgexfoundry/device-modbus-go](https://github.com/edgexfoundry/device-modbus-go/issues), and provide more information. Such as: edgex version, device profile, device definition...

Jieke
YIQISOFT

**Reply by Liguangwei0321** [reply]
thanks，i will try

**Reply by jiekechoo** [reply]
Please close this question.
